frontend/src/App.tsx

```diff
@@ -1,24 +1,21 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { Note } from './models/note';
 
 function App() {
+  const [notes, setNotes] = useState<Note[]>([]);
+  useEffect(() => {
+    async function loadNotes() {
+      const response = await fetch("/api/notes", {method: "GET"});
+      const notes = await response.json();
+      setNotes(notes);
+    }
+    loadNotes();
+  }, []);
```

Comment:
[] just call once, we can pass list variable state to [], when one of these state change then function useEffect will call again